server/broadcast.go

```diff
@@ -361,6 +361,7 @@ func transcodeSegment(cxn *rtmpConnection, seg *stream.HLSSegment, name string,
 	if sess.Sender != nil {
 		if err := sess.Sender.ValidateTicketParams(pmTicketParams(sess.OrchestratorInfo.TicketParams)); err != nil {
 			if err != pm.ErrTicketParamsExpired {
+				glog.Error("unable to submit segment err=", err)
```

Comment:
Can we update the language in this log statement to better reflect the operation that failed? Maybe something like:

```golang
glog.Error("Invalid ticket params err=", err)
```

Reply:
fixed in 571ab3d